cgroup2/utils.go

```diff
@@ -419,11 +419,17 @@ func readHugeTlbStats(path string) []*stats.HugeTlbStat {
 	hpSizes := hugePageSizes()
 	usage := make([]*stats.HugeTlbStat, len(hpSizes))
 	for idx, pagesize := range hpSizes {
+		events := make(map[string]uint64)
+		err := readKVStatsFile(path, "hugetlb."+pagesize+".events", events)
+		if err != nil {
+			log.L.WithError(err).Errorf("failed to read hugetlb events for %s", filepath.Join(path, "hugetlb."+pagesize+".events"))
+		}
```

Comment:
Should this fail hard if the file doesn't exist, or ignore non-existing (`os.IsNotExist(err)`) ? (Not sure if it's guaranteed to exist?)